ideascube/search/utils.py

```diff
@@ -12,7 +12,7 @@ def create_index_table(force=True):
         cursor_transient.execute("DROP TABLE IF EXISTS idx")
         cursor_transient.execute("CREATE VIRTUAL TABLE idx using "
                                  "FTS4(id, model, model_id, public, text, "
-                                 "lang, kind, tags)")
+                                 "lang, kind, tags, package_id)")
```

Comment:
That's a bit sad: I'd love to keep this table out of business specifics.
Can't we sort of manage this with a tag? Not saying it's a better idea, just starting the brainstorming :)